src/components/rocm_smi/README.md

```diff
@@ -21,40 +23,49 @@ Typically, the utility `papi_components_avail` (available in `papi/src/utils/pap
 ## Environment Variables
 
 For ROCM_SMI, PAPI requires one environment variable: `PAPI_ROCMSMI_ROOT`. Note
-in most installations, this is a subdirectory under the ROCM directory. This is 
-required at both compile and run time.
+that in most installations this is a subdirectory under the ROCM directory. This
+variable is required at both compile and run time.
+
+There are two common cases for setting this variable:
+
+1.  **Case 1:**  
```

Comment:
We could possibly be more explicit when Case 1 and Case 2 are needed. 

Based off their documentation, in ROCm 5.2 they restructured their filesystem to be inline with the Linux Filesystem Hierarchy Standard (https://rocm.docs.amd.com/en/latest/conceptual/file-reorg.html). Eventually in 6.0, they fully adopt the Linux Filesystem Hierarchy Standard and removed backwards compatibility. 

Based off that, Case 1 should cover ROCm < 5.2 and Case 2 should cover ROCm >= 5.2.

Reply:
Looks good to me.